### ISSUE TITLE
Is this extension still useful with Firefox 71?

### ISSUE THREAD

**Issue by rugk** [april/certainly-something#68]
It seems your add-on/the whole thing has been included in Firefox 71?

https://www.ghacks.net/2019/08/27/firefox-71-has-a-new-certificates-viewer/

So is this still useful/is there any advantage?

If yes: Could you please describe what? Maybe best also state/adjust that in the Readme, AMO description etc.
If not: Could you somehow deprecate this add-on? AMO description & Readme again. Possibly another update that just tells users where that new functionality is?
Possibly even [archive this repository](https://help.github.com/articles/archiving-repositories/), to mark it as unmaintained.

**Comment by rugk** [comment]
BTW the only difference I see is that _with_ that add-on you have 1 click to get to the certificate viewer, otherwise you need to go through a menu. :smiley: 

Maybe the add-on can be simplified to still provide that feature though then? I guess it could just open the Firefox-internal page, on click of the button?

**Comment by april** [comment]
There's not a ton of uses for the add-on anymore, other than quick access to certs and the ability to upload a certificate or certificate chain to view.  :)

**Comment by ZackBoe** [comment]
+1 to have the ability/option to direct to the built in version from the extension's address bar icon!

I really find it useful to have one click access to a site's certificate info as opposed to several clicks and dealing with the page info window - although I do love the built in version's dark mode.

I super briefly looked at creating a quick browser action extension to pop up `about:certificate`, but I'm not sure where the b64'd strings of the cert chain is from or if it's available through `webRequest.CertificateInfo`

**Comment by crssi** [comment]
I still find the extension useful because of easy access to information's.
Also the handshake is better presented.

**Comment by publicarray** [comment]
Thanks for the extension 👏 

Both the extension and inbuilt viewer in the browser uses the American date format. Can this me changed to something more universal or can it be fixed so it matches with the appropriate locale?

**Comment by rugk** [comment]
Maybe do open an issue both in… this repo… and on bugzilla, then.(?)

**Comment by publicarray** [comment]
https://bugzilla.mozilla.org/show_bug.cgi?id=1601841

**Comment by KrasnayaPloshchad** [comment]
This is still useful on Firefox for Android Nightly, if you add this extension following their [instructions](https://blog.mozilla.org/addons/2020/09/29/expanded-extension-support-in-firefox-for-android-nightly/).

**Comment by ZackBoe** [comment]
Just a heads up to anyone still keeping an eye on this looking for one click access to the built in cert viewer, `about:certificate` is considered a privileged URL and cannot yet be opened by extensions: https://bugzilla.mozilla.org/show_bug.cgi?id=1777950

**Comment by april** [comment]
Yeah, I still use it all the time for quick access to certificate information. The code from the extension was used to create the official Firefox cert viewer, so it isn't showing anything new or different.

But it is a lot fewer clicks. Hopefully one day Mozilla will make it easier to either add a direct shortcut to certificate info in your taskbar, or provide access to the internal URL for extensions.

**Comment by ZackBoe** [comment]
> so it isn't showing anything new or different.

The native viewer lacks handshake info, but has dark mode - maybe I'll throw together a PR for that :)

Folks may also find this useful, if implemented: https://bugzilla.mozilla.org/show_bug.cgi?id=1608962
